apps/server/src/modules/tldraw/tldraw-app.module.ts

```diff
@@ -0,0 +1,44 @@
+import { Dictionary, IPrimaryKey } from '@mikro-orm/core';
+import { MikroOrmModule, MikroOrmModuleSyncOptions } from '@mikro-orm/nestjs';
+import { Module, NotFoundException } from '@nestjs/common';
+import { ConfigModule } from '@nestjs/config';
+import { Course, User, School, CourseGroup, SchoolYear } from '@shared/domain';
+import { DB_PASSWORD, DB_URL, DB_USERNAME, createConfigModuleOptions } from '@src/config';
+import { CoreModule } from '@src/core';
+import { Logger } from '@src/core/logger';
+import { AuthorizationModule } from '@src/modules/authorization';
+import { AuthenticationModule } from '../authentication/authentication.module';
+import { config } from './tldraw.config';
+import { RabbitMQWrapperModule } from '@shared/infra/rabbitmq';
+import { TldrawGateway } from '@src/modules/tldraw/gateway';
+
+const defaultMikroOrmOptions: MikroOrmModuleSyncOptions = {
+	findOneOrFailHandler: (entityName: string, where: Dictionary | IPrimaryKey) =>
+		// eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+		new NotFoundException(`The requested ${entityName}: ${where} has not been found.`),
+};
```

Comment:
Are you sure you need these options as you don't really use _mikro-orm_ in the first place?